src/odemis/gui/cont/features.py

```diff
@@ -94,20 +94,23 @@ def _on_btn_go_to_feature(self, _):
         current_label = pm.getCurrentPostureLabel()
         logging.debug(f"Current posture: {POSITION_NAMES[current_label]}")
 
-        if current_label != FM_IMAGING: # TODO: @patrick remove this once SEM move is supported 
-            logging.info(f"Currently under {POSITION_NAMES[current_label]}, moving to feature position is not yet supported.")
+        # TODO: @patrick remove this once SEM move is supported
+        if current_label != FM_IMAGING and self._main_data_model.microscope.role == "meteor":
+            logging.info(f"Currently under {POSITION_NAMES[current_label]}, "
+                         f"moving to feature position is not yet supported.")
```

Comment:
@patrickcleeve2 wanted to tag you to ensure that you have a notification about it and you are indeed aware of this.

Reply:
good catch thanks